.travis.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: go
+go:
+- 1.14
+
+jobs:
+  include:
+    - stage: build
+      script:
+        - go build
+    - stage: style
+      script:
```

Comment:
did you leave gocyclo out on purpose?

Reply:
yes, for this moment - as **my** code is wrong :)